ArduCopter/Parameters.h

```diff
@@ -305,6 +305,7 @@ class Parameters {
         k_param_flight_mode4,
         k_param_flight_mode5,
         k_param_flight_mode6,
+        k_param_initial_mode,
```

Comment:
I moved this enum below k_param_flight_mode_chan,. I feel this would allocate 208 to k_param_initial_mode that was not used initially. Would this suffice